images/router/haproxy/conf/haproxy-config.template

```diff
@@ -201,7 +201,7 @@ frontend public
     {{- end }}
   {{- if isTrue (env "ROUTER_USE_PROXY_PROTOCOL") }} accept-proxy{{ end }}
   mode http
-  tcp-request inspect-delay {{ env "ROUTER_INSPECT_DELAY" "5s" }}
```

Comment:
Some of the timeout values use the `clipHAProxyTimeoutValue`, like: https://github.com/openshift/router/blob/master/images/router/haproxy/conf/haproxy-config.template#L44

Do you need to consider that?

Reply:
It shouldn't be necessary because the value is evaluated through `durationToHAProxyTimespec` when it's put into the environment variable in the first place, which acts similar to `clipHAProxyTimeoutValue`. (used here: https://github.com/openshift/cluster-ingress-operator/blob/master/pkg/operator/controller/ingress/deployment.go#L540)

That said, I think it would also make sense to weed out bad timeout values in the ingress operator